Wikipedia/Code/ArticleViewController.swift

```diff
@@ -663,7 +664,7 @@ class ArticleViewController: ThemeableViewController, HintPresenting, UIScrollVi
               let project = WikimediaProject(siteURL: siteURL) {
             
             if !isBeingPresentedAsPeek {
-                ArticleLinkInteractionFunnel.shared.logArticleView(pageID: pageID.intValue, project: project)
+                ArticleLinkInteractionFunnel.shared.logArticleView(pageID: pageID.intValue, project: project, source: articleViewSource)
```

Comment:
Can you add the source below as well, in `logPageViewAfterArticleAppearance`?

Reply:
Totally missed that, thanks for the catch!